Assets/Package/Runtime/Console/Commands/PredeterminedArgumentsCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace BroWar.Debugging.Console.Commands
+{
+    [CreateAssetMenu(fileName = "Predetermined Arguments Command", menuName = "BroWar/Debugging/Commands/Predetermined Arguments")]
+    public class PredeterminedArgumentsCommand : ConsoleCommand, IAutocompleteOptionProvider
+    {
+        public void Execute(string value)
+        {
+
+        }
+
+        void IAutocompleteOptionProvider.GetParemeterAutocompleteOptions(string[] words, int wordIndex, IList<string> optionList)
```

Comment:
Jawna implementacja interfejsu nie jest tutaj potrzebna - wystarczy public void GetParameterAutocompleteOptions

Reply:
Ale Pan Miłosz powiedział, że tak się robi ;( Na jego oczach i za jego poleceniem to pisałem.

Reply:
@arimger Czy to dlatego, że tak w Broken Roads robicie czy jest jakiś inny powód?

Reply:
To już tłumaczyłem na żywo ale ta metoda nie jest częścią publicznego API danej komendy i powinna być tylko wykorzystana przez coś co jawnie korzysta z interfejsu stąd też explicit implementacja. Na takiej samej zasadzie robi się np. callbacki z serializacji